### DISCUSSION TITLE
Renovate does not create PR when revert to older package

### DISCUSSION THREAD

**Discussion by ankitdadhich** [renovatebot/renovate#16117] [Request Help]
Recently I encountered this issue, here are steps 

1. renovate updated package to latest lets say 2.0.0
2. Due to some bug, we had to revert that commit to older version 1.0.0
3. After sometime we fixed the bug and deployed new version 3.0.0
4. Now, when renovate is run again. It does no create the PR. Because renovate assumes, it already updated the package. In the logs it just posting links to older PR, instead of creating new one. While in reality it is still outdated and using 1.0.0

How could I fix this ? Although it is outdated but renovate still does not create PR

**Comment by rarkins** [comment]
You need to look at your debug logs to know more about what's going on.

The `packageFiles with updates` message should confirm if Renovate knows it's on 1.0.0 and sees the 3.0.0 release and is aware it needs upgrading.

Then you should find the logs for the branch in question and see what those say, especially about why it is not creating the branch as you expect.

**Reply by ankitdadhich** [reply]
so it does detect the new version as per logs, but not creating new PR
`{
    "depType": "devDependencies",
    "depName": "@internal",
    "currentValue": "1.3.73",
    "datasource": "npm",
    "prettyDepType": "devDependency",
    "lockedVersion": "1.3.73",
    "depIndex": 2,
    "updates": [
        {
            "bucket": "non-major",
            "newVersion": "1.4.2",
            "newValue": "1.4.2",
            "releaseTimestamp": "2022-06-14T09:39:20.507Z",
            "newMajor": 1,
            "newMinor": 4,
            "updateType": "minor",
            "branchName": "NewBranch"
        }
    ],
    "warnings": [],
    "versioning": "npm",
    "sourceUrl": "https://gitlab.internaldev.io/xxx",
    "currentVersion": "1.3.73",
    "isSingleVersion": true,
    "fixedVersion": "1.3.73"
}
],
"packageJsonType": "app",
"npmrc": "***********",
"yarnLock": "yarn.lock",
"npmLock": "package-lock.json",
"managerData": {
"yarnZeroInstall": false,
"hasPackageManager": false
},
"skipInstalls": true,
"constraints": {},
"lockFiles": [
"yarn.lock"
]
}
]
},
"msg": "packageFiles with updates",
"time": "2022-06-17T05:05:51.580Z",
"v": 0
}`

**Reply by rarkins** [reply]
You missed the second half of my instructions
